Classes/Player/PlayerController.swift

```diff
@@ -212,10 +212,10 @@ class PlayerController: NSObject, Player, PlayerSettings {
 
 extension PlayerController {
     
-    /// Updates the request adapter if it is kaltura type
-    fileprivate func updateKalturaRequestAdapterIfExists(in mediaSource: inout MediaSource) {
-        // configure media sources content request adapter if kaltura request adapter exists
-        if let _ = self.contentRequestAdapter as? KalturaPlaybackRequestAdapter {
+    /// Updates the request adapter if one exists
```

Comment:
Actually, self.mediaSessionUUID should be set even if there's no request adapter.